Dockerfile

```diff
@@ -15,6 +15,5 @@ RUN chmod +x /bin/docker
 
 COPY . /usr/src/app
 
-ENV FILESYSTEM_DIR /code
-
+ENV CODECLIMATE_DOCKER 1
```

Comment:
Am I correct that this is only false for local dev/local tests? If that's the case, I'd suggest dropping it completely & making sure tests provide appropriate ENV. Since docker is the CLI's natural environ, special-casing for the other case (esp. if it affects tests) seems like it makes tests less honest.